src/components/services/CopyToClipboard.vue

```diff
@@ -0,0 +1,83 @@
+<template>
+  <Generic :item="item">
+    <template #indicator>
+      <div class="status"><i ref="copyIcon" class="fa-regular fa-copy fa-xl" :class="{'scale': animate}" @click="copy()"></i></div>
+    </template>
+  </Generic>
+</template>
+
+<script>
+import service from "@/mixins/service.js";
+import Generic from "./Generic.vue";
+
+export default {
+  name: "CopyToClipboard",
+  mixins: [service],
+  props: {
+    item: Object,
+  },
+  components: {
+    Generic,
+  },
+  data: () => ({
+    animate: false
+  }),
+  mounted: function() {
+    this.$refs.copyIcon.addEventListener(
+      'animationend',

```

Comment:
If you use `@animationend`, you should be able to remove this block

Reply:
True, no need for setting the listener manually.